lambdas/src/apis/collections/Utils.ts

```diff
@@ -25,8 +25,12 @@ export function preferEnglish(i18ns: I18N[]): string | undefined {
   return (i18nInEnglish ?? i18ns[0])?.text
 }
 
-export function translateEntityIntoWearable(client: SmartContentClient, entity: Entity): LambdasWearable {
-  const metadata: Wearable = entity.metadata!
+export function translateEntityIntoWearable(client: SmartContentClient, entity: Entity): LambdasWearable | undefined {
+  if (!entity.metadata || !entity.metadata.representations) {
```

Comment:
shouldn't this be `!(entity.metadata.data && entity.metadata.data.representations)` or something like that?

Reply:
That's logically the same, right?